mautic/mautic
# Change Successful Submit Action of form behaviour

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

This PR just reorder option for forms Successful Submit Action select box. 
Add Remain at form on the last possition. Also hide message input If you choose this input



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Just create form and check If
Message is first option from Successful Submit Action and selected by defaul:
![image](https://github.com/mautic/mautic/assets/462477/e65d9d19-e21c-4612-ae94-0a10ff2fd252)
3. Check also If message input is hidden of select option Remain at form

![image](https://github.com/mautic/mautic/assets/462477/54790d31-1bb2-47ee-b507-cc593184166c)
